doc/devel/testing.rst

```diff
@@ -280,7 +280,7 @@ parallelized version of tox called ``detox``. Give this a try:
 
 .. code-block:: bash
 
-    $ pip install -U -i http://pypi.testrun.org detox
+    $ pip install -U detox
```

Comment:
According to PyPI, `detox` is now unmaintained and also redundant because `tox` has a parallel option.
https://pypi.org/project/detox/